app/controllers/host_shelters/host_shelters_controller.rb

```diff
@@ -7,4 +7,11 @@ def auth_user_check
        redirect_to login_path
      end
   end
+
+  def property_owner(property_id)
+    unless current_user == Property.find(property_id).user
+      session[:user_id] = nil
```

Comment:
@erose357 Does this line log out an authorized user if they try to access a page that they don't have access to?

Reply:
I believe this was pulled into this branch on a merge.  I will double check now, but I haven't noticed a problem with it.